### DISCUSSION TITLE
Request for the FID Evaluation Code in MAISI

### DISCUSSION THREAD

**Discussion by mar-cry** [Project-MONAI/tutorials#1912] [General]
I am currently developing a CT image generation algorithm and would like to compare its performance with the results reported in your work, MAISI. However, we have observed a significant discrepancy between the FID values calculated using our code and those reported in your paper.

To ensure fairness in the comparison, I would like to kindly request the code or implementation details used in MAISI for calculating the FID between generated and real images. This would greatly help in maintaining consistency and accuracy in our evaluations.

**Comment by paproaskwar** [comment]
Which part of body your algorithm generate?

**Comment by KumoLiu** [comment]
Replied here: https://github.com/Project-MONAI/MONAI/discussions/8243#discussioncomment-11430253
(move to discussion for now)
